statsmodels/tsa/vector_ar/irf.py

```diff
@@ -228,6 +233,7 @@ def plot_cum_effects(self, orth=False, impulse=None, response=None,
                                      hlines=lr_effects,
                                      subplot_params=subplot_params,
                                      plot_params=plot_params,
+                                     figsize=figsize,
```

Comment:
Need a test.

Reply:
Finally, I have included one test specifically for the figure size. Please let me know if there is any other changes I can make.